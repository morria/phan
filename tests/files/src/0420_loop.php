<?php

/**
 * @return bool
 */
function foreach420(stdClass $params) {
    foreach ($params->results as $reviewItem) {
        // NOTE: PHP would have already resolved $params->results - This should not cause a warning above.
        $params = [ ];
    }
    return true;
}
