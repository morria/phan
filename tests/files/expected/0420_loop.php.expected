%s:7 PhanUnusedVariable Unused definition of variable $reviewItem
%s:9 PhanUnusedVariable Unused definition of variable $params
